src/nft/utils/listNfts.ts

```diff
@@ -59,7 +59,7 @@ const getConsiderationItems = (
   openSeaFee?: ConsiderationInputItem
 } => {
   const creatorFeeBasisPoints = asset?.basisPoints ?? 0
-  const openSeaBasisPoints = !asset?.basisPoints ? 50 : 0
+  const openSeaBasisPoints = (ListingMarkets.find((market) => market.name === 'OpenSea')?.fee ?? 0) * 100
```

Comment:
Previously OS would require a 50 basis point fee for collections with no creator royalties and 0 basis points if it had royalties but now it's just the flat 250 basis points